Missing information for plugin creation #834 (#836)
strapi/documentation: docs/developer-docs/latest/development/plugins-development.md

      // ...
    }
    ```

>     * The plugin name you use as key must be kebab-cased, other casing will not work.

Reviewer: ```suggestion
    ::: caution 
    The plugin name must be kebab-case.
    Example: an-example-of-kebab-case
    :::
```
```suggestion
    * The plugin name you use as key must be kebab-cased, other casing will not work.
```
